guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc

```diff
@@ -1,9 +1,14 @@
-[id="synchronizing-an-external-{smart-proxy-context}_{context}"]
-= Synchronizing an External {SmartProxy}
+[id="Synchronizing_a_{smart-proxy-context-titlecase}_Server_{context}"]
+= Synchronizing a {SmartProxyServer}
```

Comment:
```suggestion
= Synchronizing content from {ProjectServer} to {SmartProxyServers}
```

Should I change the title/anchor/file name?

Reply:
Sure.

Reply:
Done!